src/thread.rs

```diff
@@ -0,0 +1,9 @@
+use decode_files::{LogFormat, processLine};
+use message::Message;
+
+const FORMAT: LogFormat = LogFormat::Textual1;
+
+fn thread(line: &str) -> Result<Vec<Data>, MessageFormatException> {
```

Comment:
does this have to be named thread? Feels like a bad name

Reply:
i have no idea what this is

Reply:
I think this is for reading things from the SD Card which I had nothing to do with, thats a peter thing, idek if this works